routes/hydrology.py

```diff
@@ -264,19 +377,19 @@ def run_get_hydrology_point_data(lat, lon):
         else:
             return render_template("400/bad_request.html"), 400
 
-    elif "summarize" in request.args:
-        if request.args.get("summarize") == "mmm":
-            try:
-                return run_fetch_hydrology_point_data_mmm(lat, lon)
-            except Exception as exc:
-                if hasattr(exc, "status") and exc.status == 404:
-                    return render_template("404/no_data.html"), 404
-                return render_template("500/server_error.html"), 500
+    elif "summarize" in request.args or summarize:
+        if request.args.get("summarize") == "mmm" or summarize:
+            # try:
+            return run_fetch_hydrology_point_data_mmm(lat, lon, summarize)
+            # except Exception as exc:
+            #     if hasattr(exc, "status") and exc.status == 404:
+            #         return render_template("404/no_data.html"), 404
+            #     return render_template("500/server_error.html"), 500
```

Comment:
Uncomment this guy I think :)

Reply:
Thanks for catching this Charlie! I've got the try-catch block uncommented now.